### DISCUSSION TITLE
Should we pass sentences that contain no entities when training NER model?

### DISCUSSION THREAD

**Discussion by erotavlas** [explosion/spaCy#3954] [Help: Best practices]
Is it ok to pass sentences that contain no entities when training the NER model?  Will spacy even accept these sentences and learn from them, or will it discard them?

**Comment by ines** [comment]
Yes, absolutely – in fact, it's super important. The model shouldn't only be seeing texts that *have* entities, but also texts that have *no* entities (i.e. an empty list). Otherwise, it may struggle to deal with those inputs at runtime.
